src/main/java/frc/trigon/robot/subsystems/intake/IntakeCommands.java

```diff
@@ -31,15 +31,16 @@ public static Command getSetTargetVoltageCommand(double targetVoltage) {
         );
     }
 
-    private static Command getCollectionCommand() {
+    private static Command getCollectionCommand(boolean isAutonomous) {
         return new SequentialCommandGroup(
                 new FunctionalCommand(
                         () -> RobotContainer.INTAKE.setTargetState(IntakeConstants.IntakeState.COLLECT),
                         () -> {
                         },
                         (interrupted) -> {
                             if (!interrupted) {
-                                RobotContainer.INTAKE.indicateCollection();
+                                if (!isAutonomous)
+                                    RobotContainer.INTAKE.indicateCollection();
                                 RobotContainer.INTAKE.sendStaticBrakeRequest();
                             }
                         },
```

Comment:
Make a seperate command for autonomous and put it into `AutonomousCommands`

Reply:
I can't put it in AutonomousCommands without making a ton of methods public which I don't like, so I can either leave it the way it is, or make a getAutonomousCollectionCommand method in this class which I also don't like.

Reply:
@levyishai what's your opinion on this?